test/webhook-test-proxy/Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR /webhook-test-proxy
 
 # Build
 COPY . .
-RUN go build -a -o webhook-test-proxy cmd/webhook-test-proxy.go
+RUN go build -ldflags="-s -w" -a -o webhook-test-proxy cmd/webhook-test-proxy.go
```

Comment:
how come this is needed for `go1.15` now?

Reply:
it's not needed, but I've been meaning to do it for a while, so I stuck it in here :D